concepticondata/conceptlists/Bowern-2021-207a.tsv

```diff
@@ -0,0 +1,208 @@
+Bowern-2021-207a-69	69	fear	412	mental	Basic Vocabulary	781	FEAR (FRIGHT)
+Bowern-2021-207a-70	70	feather	413	fauna	Basic Vocabulary	1201	FEATHER
+Bowern-2021-207a-71	71	fingernail	821	body	Basic Vocabulary	1258	FINGERNAIL
+Bowern-2021-207a-72	72	fire	414	environment	Basic Vocabulary	221	FIRE
+Bowern-2021-207a-73	73	fish	415	fauna	Basic Vocabulary	227	FISH
+Bowern-2021-207a-74	74	flow	416	motion	Basic Vocabulary	2003	FLOW
+Bowern-2021-207a-75	75	flower	278	flora-fauna	Basic Vocabulary	239	FLOWER
+Bowern-2021-207a-76	76	flower	417	environment	Basic Vocabulary	239	FLOWER
+Bowern-2021-207a-77	77	fly	418	motion	Basic Vocabulary	1441	FLY (MOVE THROUGH AIR)
+Bowern-2021-207a-78	78	fog	419	environment	Basic Vocabulary	249	FOG
+Bowern-2021-207a-79	79	foot	420	body	Basic Vocabulary	1301	FOOT
+Bowern-2021-207a-80	80	fruit	421	flora	Basic Vocabulary	1507	FRUIT
+Bowern-2021-207a-81	81	good	422	quality	Basic Vocabulary	1035	GOOD
+Bowern-2021-207a-82	82	green	825	quality	Basic Vocabulary	1425	GREEN
+Bowern-2021-207a-83	83	grow	424	state	Basic Vocabulary	479	GROW
+Bowern-2021-207a-84	84	hair (of head)	425	body	Basic Vocabulary	2648	HAIR (HEAD)
+Bowern-2021-207a-85	85	hand	426	body	Basic Vocabulary	1277	HAND
+Bowern-2021-207a-86	86	head	428	body	Basic Vocabulary	1256	HEAD
+Bowern-2021-207a-87	87	hear	429	mental	Basic Vocabulary	1408	HEAR
+Bowern-2021-207a-88	88	heart	822	body	Basic Vocabulary	1223	HEART
+Bowern-2021-207a-89	89	heavy	430	quality	Basic Vocabulary	1210	HEAVY
+Bowern-2021-207a-90	90	hide	431	motion	Basic Vocabulary	602	HIDE (CONCEAL)
+Bowern-2021-207a-91	91	hit	432	impact	Basic Vocabulary	1433	HIT
+Bowern-2021-207a-92	92	hit (with hand)	576	impact	Basic Vocabulary		
+Bowern-2021-207a-93	93	hold	433	other	Basic Vocabulary	1448	HOLD
+Bowern-2021-207a-94	94	hot	826	quality	Basic Vocabulary	1286	HOT
+Bowern-2021-207a-95	95	how?	435	grammar	Basic Vocabulary	1239	HOW
+Bowern-2021-207a-96	96	if	437	grammar	Basic Vocabulary	1459	IF
+Bowern-2021-207a-97	97	in/inside	438	location	Basic Vocabulary	1606	INSIDE
+Bowern-2021-207a-98	98	intestines	439	body	Basic Vocabulary	1991	INTESTINES
+Bowern-2021-207a-99	99	itch	440	body	Basic Vocabulary	3855	ITCH (CAUSE ITCHING OR FEEL ITCHY)
+Bowern-2021-207a-100	100	kill	442	impact	Basic Vocabulary	1417	KILL
+Bowern-2021-207a-101	101	know/be knowledgeable	579	mental	Basic Vocabulary	1410	KNOW (SOMETHING)
+Bowern-2021-207a-102	102	lake	444	environment	Basic Vocabulary	624	LAKE
+Bowern-2021-207a-103	103	laugh	445	mental	Basic Vocabulary	1355	LAUGH
+Bowern-2021-207a-104	104	leaf	446	environment	Basic Vocabulary	628	LEAF
+Bowern-2021-207a-105	105	left/left hand	447	location/body	Basic Vocabulary		
+Bowern-2021-207a-106	106	leg/foot	580	body	Basic Vocabulary	2098	FOOT OR LEG
+Bowern-2021-207a-107	107	lie down	449	motion	Basic Vocabulary	215	LIE DOWN
+Bowern-2021-207a-108	108	lightning	450	environment	Basic Vocabulary	1654	LIGHTNING
+Bowern-2021-207a-109	109	live/be alive	451	body	Basic Vocabulary	2309	LIVE
```

Comment:
This is usually mapped to BE ALIVE.

Reply:
In the same list, there already is "be alive". Still map this one to BE ALIVE as well or unmap?

Reply:
Just saw that LIVE is a duplicate. I'd leave it unmapped then.

Reply:
Good catch!